sklearn/cluster/_dbscan.py

```diff
@@ -156,18 +156,18 @@ class DBSCAN(ClusterMixin, BaseEstimator):
 
     Parameters
     ----------
-    eps : float, optional
+    eps : float, default=0.5
         The maximum distance between two samples for one to be considered
         as in the neighborhood of the other. This is not a maximum bound
         on the distances of points within a cluster. This is the most
         important DBSCAN parameter to choose appropriately for your data set
         and distance function.
 
-    min_samples : int, optional
+    min_samples : int, default=5
         The number of samples (or total weight) in a neighborhood for a point
         to be considered as a core point. This includes the point itself.
 
-    metric : string, or callable
+    metric : string, default='euclidean'
```

Comment:
This can still be a callable.

Reply:
Yes, thank you. I have just updated the doc with "or callable"